assets/armsnippets.jsonc

```diff
@@ -1167,16 +1165,16 @@
             "{",
             "    \"name\": \"${1:aksCluster1}\",",
             "    \"type\": \"Microsoft.ContainerService/managedClusters\",",
-            "    \"apiVersion\": \"2018-03-31\",",
+            "    \"apiVersion\": \"2020-02-01\",",
             "    \"location\": \"[resourceGroup().location]\",",
             "    \"properties\": {",
-            "        \"kubernetesVersion\": \"${2:1.9}\",",
-            "        \"dnsPrefix\": \"$32:dnsPrefix}\",",
+            "        \"kubernetesVersion\": \"${2|1.15.7,1.15.5,1.14.8|}\",",
+            "        \"dnsPrefix\": \"${3:dnsprefix}\",",
             "        \"agentPoolProfiles\": [",
             "            {",
             "                \"name\": \"agentpool\",",
             "                \"count\": ${4:2},",
-            "                \"vmSize\": \"${5:virtualMachineSize}\",",
+            "                \"vmSize\": \"${5:Standard_A1}\",",

```

Comment:
Is it worth it to put in common values here?  I know they add a lot frequently, though, so maybe not??

Reply:
It's a pretty big list. I had considered putting in a few, however did not want to give the impression that the value was limited to that set.